docs/development/releases-and-deployment.md

```diff
@@ -0,0 +1,102 @@
+# Spark release / deployment guidelines
+
+## GitHub releases
+
+* https://github.com/Midburn/Spark/releases
+* There should always be 1 draft release against master branch
+* This draft release is updated on each merge or commit to master.
+
+### Publishing a release
+
+* When release is ready (e.g. have enough content / is stable enough)- you can publish it
+* **Before publishing on GitHub** - you should update the [package.json](/package.json) "version" attribute to match the GitHub release you are about to publish
+* merge the change in package.json to master
+* don't forget to push!
+* now, you can publish the release in GitHub
+
+## Manual Deployment
+
```

Comment:
I see now you wrote below "basically we want as much work done here as possible so deployment is quick" which touches on this a bit.